### PR TITLE
INHEART-142: Modify & slightly refactor Icon atom component and add new icons

### PR DESCRIPTION
**Pull request by pawelroguski**
- Create posibility to get specific icon by passing icon's name by component props;
- Delete old SignIn page (apps/mobile/src/app/pages/SignIn);
- Add new icons;
- Add tests for icons;
- Add storybook;

### CODE REVIEW

File: libs/ui/atoms/src/lib/Icons/Icons.tsx [hunk 1]
```diff
@@ -1,3 +1,212 @@
-import { WarningOutlineIcon } from 'native-base';
```
Comment (konradhanus) [inline libs/ui/atoms/src/lib/Icons/Icons.tsx:1]: - [ ] - add storybook for this file

**Review by konradhanus — APPROVED** [review]

File: libs/ui/atoms/src/lib/Icons/__stories__/Icons.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
+//TODO - this stories is not visible in StoryBook
```
Comment (konradhanusintive) [inline libs/ui/atoms/src/lib/Icons/__stories__/Icons.stories.tsx:1]: remove it

Comment (KonradKarlinski) [inline libs/ui/atoms/src/lib/Icons/__stories__/Icons.stories.tsx:1]: Remove it

File: libs/ui/atoms/src/lib/Icons/__tests__/Icons.spec.tsx [hunk 3]
```diff
@@ -1,14 +1,46 @@
...
+    'three-dots',
+    'share',
+    'play',
+    'favourite',
+    'delete',
+  ]).test('should render successfully all icons', (iconName) => {
     const { container } = render(
       <NativeBaseProvider>
-        <WarningOutlineIcon />
+        <Icons iconName={iconName} />
```
Comment (konradhanusintive) [inline libs/ui/atoms/src/lib/Icons/__tests__/Icons.spec.tsx:43]: this test is always true

**Comment by BeataKula** [comment]
Add screenshots or something like that to the description

File: libs/ui/atoms/src/lib/Icons/Icons.tsx [hunk 4]
```diff
@@ -1,3 +1,248 @@
...
+    | 'share'
+    | 'play'
+    | 'favourite'
+    | 'delete'
+    | 'star'
+    | 'star-outline'
+    | 'home'
+    | 'home-outline'
+    | 'plus-circle';
+}
```
Comment (BeataKula) [inline libs/ui/atoms/src/lib/Icons/Icons.tsx:78]: As I remember it should be change to enum type.

File: libs/ui/atoms/src/lib/Icons/__stories__/Icons.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+  'play',
+  'favourite',
+  'delete',
+  'star',
+  'star-outline',
+  'home',
+  'home-outline',
+  'favorite',
+  'favorite-outline',
+  'plus-circle-outline',
```
Comment (BeataKula) [inline libs/ui/atoms/src/lib/Icons/__stories__/Icons.stories.tsx:44]: Maybe it would be better to export this icon Array and use it on Icons.spec.tsx?

**Review by BeataKula — APPROVED** [review]

**Review by konradhanusintive — APPROVED** [review]

**Pull request merged**